File: ViewModelsLayer/ViewModels/HistoryViewModels/ResponseGameDetailsViewModel.cs
using System.Collections.Generic;

namespace ViewModelsLayer.ViewModels.HistoryViewModels
{
    public class ResponseGameDetailsViewModel
    {
        public List<ResponseRoundDetailsViewModel> Rounds { get; set; }
        public List<HistoryGameStatisticViewModel> Statistic { get; set; }
    }
}

File: API/Angular/src/app/viewmodels/HistoryViewModels/response-game-details-view-model.ts
import ResponseRoundDetailsViewModel from './response-round-details-view-model';
import HistoryGameStatisticViewModel from './history-game-statistic-view-model';

export default class ResponseGameDetailsViewModel {
  constructor(rounds, statistic) {
    this.rounds = rounds;
    this.statistic = statistic;
  }

  public rounds: Array<ResponseRoundDetailsViewModel>;
  public statistic: Array<HistoryGameStatisticViewModel>;
}
